scripts/west_commands/runners/core.py

```diff
@@ -280,6 +280,12 @@ def __call__(self, parser, namespace, values, option_string=None):
             namespace.dt_flash = False
 
 
+class _ToggleAction(argparse.Action):
```

Comment:
This is copy/pasted from sign.py. I considered making this shared code, but the runners package is meant to be standalone code, and "a little copying is better than a little dependency" in this case IMO.